test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -154,6 +156,16 @@ var _ = BeforeSuite(func() {
 		dnsAddonName = common.CoreDNSAddonName
 	}
 	Expect(dnsAddonName).NotTo(Equal(""))
+
+	var client *armhelpers.AzureClient
+	env, err := azure.EnvironmentFromName("AzurePublicCloud") // TODO get this programmatically
```

Comment:
@jadarsie remind me, are you using this test runner in Azure stack? If so we might want to skip this new foo in Azure Stack contexts